system-parachains/coretime/coretime-kusama/src/coretime.rs

```diff
@@ -253,8 +219,6 @@ impl pallet_broker::Config for Runtime {
 	type WeightInfo = weights::pallet_broker::WeightInfo<Runtime>;
 	type PalletId = BrokerPalletId;
 	type AdminOrigin = EnsureRoot<AccountId>;
-	#[cfg(feature = "runtime-benchmarks")]
-	type PriceAdapter = pallet_broker::Linear;
-	#[cfg(not(feature = "runtime-benchmarks"))]
-	type PriceAdapter = PriceAdapter;
+	type PriceAdapter = pallet_broker::CenterTargetPrice<Balance>; // FAIL-CI @donal please bring back the custom PriceAdapter. I remove all the tests for the
+															   // custom PriceAdapter as well, probably needs to be reverted if you still want to use it.
```

Comment:
@seadanda i had to temporarily delete the PriceAdapter to make it compile since the API changed, please bring it back - if needed.

Reply:
I just saw that it was remove on Master. So deleting the `PriceAdapter` is fine as well?

Reply:
Yes the `PriceAdapter` now uses an impl defined in the broker pallet (this CenterTargetPrice) for the new price model that is being tested on Kusama - we'll recreate the impl directly in the fellows repo later so it's under the control of the fellowship but for now it's fine to delete and use this.

Reply:
I think re-creation only makes sense once you have something you actually want to change.